### ISSUE TITLE
Event error resulting from problematic URI in sonos queue

### ISSUE THREAD

**Issue by relevitt** [SoCo/SoCo#380]
If you try to add a spotify item to the queue as follows:

```
uri = 'x-sonos-spotify:spotify%3Atrack%3A46OYzA4Mp14jbIK0Pdy6hH?sid=9&flags=8224'
device.add_uri_to_queue(uri)
```

and then you try to play the item, and you're subscribed to events via the soco.events module,
then the following error may occur:

```
----------------------------------------
Exception happened during processing of request from ('192.168.1.85', 35347)
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 593, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python2.7/SocketServer.py", line 649, in __init__
    self.handle()
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 340, in handle
    self.handle_one_request()
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 328, in handle_one_request
    method()
  File "/usr/local/lib/python2.7/dist-packages/soco/events.py", line 230, in do_NOTIFY
    variables = parse_event_xml(content)
  File "/usr/local/lib/python2.7/dist-packages/soco/events.py", line 136, in parse_event_xml
    value = from_didl_string(value)[0]
  File "/usr/local/lib/python2.7/dist-packages/soco/data_structures.py", line 82, in from_didl_string
    items.append(cls.from_element(elt))
  File "/usr/local/lib/python2.7/dist-packages/soco/data_structures.py", line 389, in from_element
    title = really_unicode(title_elt.text)
  File "/usr/local/lib/python2.7/dist-packages/soco/utils.py", line 30, in really_unicode
    raise ValueError('%s is not a string at all.' % in_string)
ValueError: None is not a string at all.
----------------------------------------
```

No doubt, an answer is not to try to add items to the sonos queue in that way, but, as you can do it, the events module should probably be able to cope.

**Comment by bosim** [comment]
I get this error even when my queue does not hold elements from Spotify. Hope this error will be fixed soon, the event system is pretty useless without a fix.

**Comment by robwebset** [comment]
Hi @bosim 
The pull request you submitted #409 - would it close this bug?
Thanks
Rob

**Comment by bosim** [comment]
@robwebset I think so. I needed two bugfixes to get the event system running perfectly, both for transport, rendering, queue. I haven't commented on the pullreq yet, been busy with other stuff.

**Comment by ghcs27** [comment]
This was (hopefully) resolved by #467. If it still doesn't work for you, feel free to reopen.
